src/Features/CSharpTest/AddUsing/AddUsingTestsWithAddImportDiagnosticProvider.cs

```diff
@@ -145,16 +128,6 @@ public async Task TestMalformedGenericParameters()
 class Class
 {
     List< }");
-
```

Comment:
Where did this go?

Reply:
It was moved to another class and it was renamed to still support this scenario